test/verify/check-storage-luks

```diff
@@ -33,6 +33,7 @@ def console_screenshot(machine, name):
 
 
 @testlib.nondestructive
+@testlib.skipImage("cryptsetup uses too much memory, OOM on our test VMs", "rhel-8-*")
```

Comment:
Did they change the LUKS encryption to argon2? (As far as I can see it still uses aes, so why is it a problem).

Reply:
Nevermind there is a jira issue about this.